docs/monitoring/01-01-monitoring.md

```diff
@@ -12,3 +12,5 @@ The whole installation package provides the end-to-end Kubernetes cluster monito
 - Set up channels for notifications informing of any detected alerts.
 
 >**NOTE:** The monitoring component is available by default in the cluster installation, but disabled in the **Kyma Lite** local installation on Minikube. [Enable the component](/root/kyma/#configuration-custom-component-installation-add-a-component) to install it with the [local profile](/components/monitoring/#configuration-monitoring-profiles-local-profile).
+
+You can install Monitoring as part of Kyma predefined [profiles](/root/kyma/#installation-overview-profiles). For production purposes, use the **production profile** which ensures increased retention time to prevent data loss in case of prolonged troubleshooting, and increased memory and CPU values to ensure stable performance of the component.
```

Comment:
I like the suggestion to provide a list, thanks! 🙇 However, I'd stay with `in case of` as here it means `if the prolonged troubleshooting happens`. Leaving [this](https://jakubmarian.com/in-the-case-vs-in-case-in-english/) article here for reference.

Reply:
If you mean `when troubleshooting suddenly happens` that's fine. I suggested `in the case of` because I understood it as `for this particular situation when you troubleshoot sth`. So, it simply depends on what message you want to convey in this sentence. :)